### DISCUSSION TITLE
[new Feature] Pull Request shows an activity log when referenced by a release

### DISCUSSION THREAD

**Discussion by Eomm** [community/community#76351] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hello,
when we write a Pull Request link into an issue, the Pull Request shows a nice activity log:

<img width="939" alt="image" src="https://github.com/community/community/assets/11404065/e8aa2071-c274-4e67-88cb-1f570312bf1f">

The "Generate release notes" button creates a perfect list of PRs in the release message:

<img width="784" alt="image" src="https://github.com/community/community/assets/11404065/0e33ba51-1e6a-40d4-9b45-a4cd1193edac">

Here is the Feature Request:

### A Pull Request should show a new activity log when it is referenced by a Release.

**Comment by ruudk** [comment]
This makes so much sense!
